Parse/src/test/java/com/parse/ParseClientConfigurationTest.java

```diff
@@ -22,9 +32,21 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.robolectric.Shadows.shadowOf;
 
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE, packageName = "com.parse.example")
```

Comment:
line 39 is needed to setup Robolectric environment
line 40 yeah, configuring `packageName` is redundant (cat just use the default value)
If `manifest` is not configured, the following log is printed during the test.
```
WARNING: No manifest file found at ./AndroidManifest.xml.
Falling back to the Android OS resources only.
To remove this warning, annotate your test class with @Config(manifest=Config.NONE).
```
But not affect the test.
It seems more appropriate using `@Config(constants = BuildConfig.class)` instead. (http://robolectric.org/getting-started/)
Robolectric will attempt to configure using project's manifest.
Thanks!